mne/viz/raw.py

```diff
@@ -299,10 +299,45 @@ def plot_raw(raw, events=None, duration=10.0, start=0.0, n_channels=20,
     # plot event_line first so it's in the back
     event_lines = [params['ax'].plot([np.nan], color=event_color[ev_num])[0]
                    for ev_num in sorted(event_color.keys())]
+
     params['plot_fun'] = partial(_plot_raw_traces, params=params, inds=inds,
                                  color=color, bad_color=bad_color,
                                  event_lines=event_lines,
                                  event_color=event_color)
+
+    if hasattr(raw, 'annotations'):
```

Comment:
I think it would be better to modify `_BaseRaw.__init__` to set `self.annotations = None`. Then this can just be `if raw.annotations is not None:`